README.md

```diff
@@ -67,6 +71,11 @@ image = Visualizer(Style.Dark).to_svg(wiretrace, start=0, length=8)
 
 # Display to stdout:
 image.display()
+
+# Manually convert EVCD file to VCD file:
+with open('myevcd.evcd', 'rb') as stream:
+    vcd = evcd2vcd(stream)
+    open('myvcd.vcd', 'wb').write(vcd.read())
```

Comment:
Make sure to close the file after it's opened.

Reply:
Now it's wrapped in `with`